README.md

```diff
@@ -11,6 +11,9 @@ SciTran Core is a RESTful HTTP API, written in Python and backed by MongoDB. It
 
 ### [Documentation](https://scitran.github.io/core)
 
+API documentation for branches and tags can be found at https://scitran.github.io/branches/**branchname** and
+https://scitran.github.io/tags/**tagname**.
+
```

Comment:
The links are missing a `/core` particle and the `**` renders weird. Maybe try angle brackets?